code/src/java/pcgen/cdom/facet/analysis/LoadFacet.java

```diff
@@ -131,7 +131,7 @@ public Float getMaxLoad(CharID id, double mult)
 	 */
 	private double getLoadMultForSize(CharID id)
 	{
-		SizeAdjustment sadj = sizeFacet.get(id);
+		SizeAdjustment sadj = pcFacet.getPC(id).getSizeAdjustment();
```

Comment:
Is there a guarantee of getPC not returning null? Otherwise this is a potential NPE.

I would suggest adding a check.

Reply:
Guaranteed non null

Reply:
Not worth a discussion here, but might be nice to pepper the source with `@Nullable`.